src/main/java/ui/PanelMenuCreator.java

```diff
@@ -48,15 +45,24 @@ public Menu generatePanelMenu() {
         items.add(createLeftPanelMenuItem());
         items.add(createRightPanelMenuItem());
         items.add(closePanelMenuItem());
-        autoCreateItems.add(createCustomizedPanelMenuItem(ASSIGNEE_PANEL_NAME, ASSIGNEE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(MILESTONE_PANEL_NAME, MILESTONE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(UPDATED_PANEL_NAME, UPDATED_FILTER_NAME));
+        for (Map.Entry<String, String> entry : generateCustomizedPanelDetails().entrySet()) {
+            autoCreateItems.add(createCustomizedPanelMenuItem(entry.getKey(), entry.getValue()));
+        }
         autoCreatePanelMenu.getItems().addAll(autoCreateItems);
         panelMenu.getItems().addAll(items);
         panelMenu.getItems().add(autoCreatePanelMenu);
         return panelMenu;
     }
 
+    public Map<String, String> generateCustomizedPanelDetails(){
```

Comment:
You can move the last login username into a parameter. That makes the function signature a bit clearer -- instead of taking the whole context of this object only to call one method, it takes the result of that method directly.